dashboard/app/views/levels/show.html.haml

```diff
@@ -28,6 +28,32 @@
   -# Render the hidden stage partial, but make it hidden so that client can unhide if needed
   = render partial: 'levels/hidden_stage', locals: {hidden: true}
 
+  - if @game.unplugged?
+    :javascript
+      var appOptions = #{app_options.to_json};
+    = render partial: 'levels/unplug', locals: {app: @game.app}
+  - elsif (@level.is_a?(Applab) || @level.is_a?(Gamelab) || @level.is_a?(Weblab)) && app_options['readonlyWorkspace'] && !app_options['channel']
+    -# A teacher is trying to view a student's progress on a level they have not started.
+    = render partial: 'levels/not_started'
+  - elsif @level.is_a?(Blockly) && @level.embed == 'true' && !@edit_blocks && !@level.is_a?(Applab) && !@level.is_a?(Gamelab) && !@level.is_a?(Weblab)
+    #title
+      - if @level.properties['short_instructions'].present?
+        %h4{style: 'margin-left: auto; margin-right: auto; max-width: 800px;'}!= brakeman_no_warn(@level.properties['short_instructions'])
+    -# Pass special parameters to Blockly partial for embedded level type.
+    = render partial: 'levels/blockly'
+  - elsif @level.is_a?(DSLDefined) || @level.is_a?(Widget) || @level.is_a?(FreeResponse) || @level.is_a?(StandaloneVideo) || @level.is_a?(ExternalLink) || @level.is_a?(CurriculumReference)
+    :javascript
+      var appOptions = #{app_options.to_json};
+      dashboard.createCallouts(appOptions.callouts);
+    - if @level.is_a?(Widget)
+      = render partial: 'levels/widget'
+    - else
+      = render partial: "levels/#{@level.class.to_s.underscore}"
+    - unless @level.is_a?(StandaloneVideo) || @level.properties["hide_reference_area"]
```

Comment:
This seems like the only line here that is specific to the video player. What is the rest of this change to this file doing?

Reply:
same question asked a different way... much of this looks duplicated from the block starting with `- if @game.unplugged?` on line 72. is there a reason we can't add a single new condition there?

Reply:
merge conflict gone sour, detangling now... thanks for catching! you're right I just want the line that checks for `hide_reference_area`.